README.md

```diff
@@ -263,6 +263,10 @@ headers.
 | SUMMARY  | com.google.ads.googleads.lib.request.summary | INFO          | WARN          |
 | DETAIL   | com.google.ads.googleads.lib.request.detail  | DEBUG         | INFO          |
 
+**Caveat**: Partial failures does not cause the entire request to fail if some of its operations fail.
```

Comment:
Readers may not know what partial failure is, so I suggest changing this to:

**Caveat**: Mutate requests where [`partialFailure`](https://developers.google.com/google-ads/api/reference/rpc/google.ads.googleads.v1.services) is `true` will not fail if individual operations fail.